server/main.lua

```diff
@@ -343,3 +343,8 @@ ESX.RegisterServerCallback('esx_mechanicjob:getPlayerInventory', function(source
 
 	cb({items = items})
 end)
+
```

Comment:
Wrong implementation. Check Line 186 of client/main.lua

https://github.com/lockdownstudio/esx_mechanicjob/blob/1da4cceee7f6424a6f1ad7e74fbc27af1bc3e7af/client/main.lua#L186

Reply:
Updated implementation